resources/js/index.scss

```diff
@@ -12,4 +12,32 @@
   input[type="checkbox"]:disabled:not(:checked) {
     filter: brightness(0.8);
   }
+
+  button {
+    &.primary {
+      @apply p-2 rounded bg-cyan-500 text-white disabled:bg-opacity-50;
+    }
+    &.secondary {
+      @apply p-2 rounded border border-cyan-500 text-cyan-500 disabled:bg-opacity-50;
+    }
+  }
+
+  // Icons
+  svg {
+    // An icon displayed inline in text. It is black by default, and reacts to text-opacity-* classes.
+    // Use the dimmed class to dim.
+    &.inlineIcon {
+      @apply inline mb-1 fill-current;
+      &.strokeIcon {
```

Comment:
Jeszcze jedna uwaga stylistyczna. W tailwindzie wszystko-jest-tak, a tu nagleMamyTak. Przydałoby się może-jednak-mieć-tak, bo trochę dziwnie potem w użyciu te dwa różne case'y wyglądają zestawione

Reply:
Ale nie wydaje mi się w sumie że to wielki problem że się nasze klasy odróżniają (przynajmniej czasem) od wbudowanych klas które na dodatek nie są prawdziwymi funkcyjnymi klasami tylko skrótem CSSa. A zresztą w plikach .module.scss i tak trzeba camelCase bo potem się tego używa w tsx po kropce. Więc bym przyjął że nasze klasy są camelCase i trudno.